src/vm/arm64/asmhelpers.asm

```diff
@@ -184,18 +184,18 @@ Done
 ; The call in ndirect import precode points to this function.
         NESTED_ENTRY NDirectImportThunk
 
-        PROLOG_SAVE_REG_PAIR           fp, lr, #-160!
+        PROLOG_SAVE_REG_PAIR           fp, lr, #-240!
```

Comment:
You've forgotten to change the 240->224 in this file too.

Reply:
@janvorli - thanks! I couldn't figure this out at first because I had "Asmhelpers.asm" open in VS, and it was showing the `240` offset - but it was the one under the obj directory, so changing that one was not useful :-(
Fixed now.